org.eclipse.cdt.cross.arc.gnu/plugin.xml

```diff
@@ -1697,6 +1765,15 @@
                name="Omit all symbol information (-s)"
                valueType="boolean">
          </option>
+         <option
+               category="org.eclipse.cdt.cross.arc.gnu.cpp.link.category.general"
+               command="--specs=nano.specs"
+               defaultValue="false"
+               id="org.eclipse.cdt.cross.arc.gnu.cpp.link.option.nano"
+               name="Use nano.specs"
+               superClass="org.eclipse.cdt.cross.arc.gnu.base.option.debugging.other.nano"
+               valueType="boolean">
```

Comment:
Most of those attributes are already specified in the super class, is there a need to repeat them?